store/store_test.go

```diff
@@ -221,6 +221,10 @@ func (t *remoteRepoTestSuite) SetUpTest(c *C) {
 	dirs.SetRootDir(c.MkDir())
 	c.Assert(os.MkdirAll(dirs.SnapMountDir, 0755), IsNil)
 
+	oldSnapdDebug := os.Getenv("SNAPD_DEBUG")
```

Comment:
AFAIR we should not use setenv much as it is unreliable in multi threaded programs. Could we mock something else to control the effective value of "SNAPD_DEBUG" in tests?